tsconfig.base.json

```diff
@@ -21,10 +21,12 @@
       "@sagebionetworks/agora/api-client-angular": ["libs/agora/api-client-angular/src/index.ts"],
       "@sagebionetworks/agora/config": ["libs/agora/config/src/index.ts"],
       "@sagebionetworks/agora/models": ["libs/agora/models/index.ts"],
+      "@sagebionetworks/agora/news": ["libs/agora/news/src/index.ts"],
       "@sagebionetworks/agora/not-found": ["libs/agora/not-found/src/index.ts"],
       "@sagebionetworks/agora/services": ["libs/agora/services/src/index.ts"],
       "@sagebionetworks/agora/teams": ["libs/agora/teams/src/index.ts"],
-      "@sagebionetworks/agora/testing/mocks": ["libs/agora/testing/src/lib/mocks"],
+      "@sagebionetworks/agora/testing/mocks": ["libs/agora/testing/src/lib/mocks/index.ts"],
+      "@sagebionetworks/agora/testing/server": ["libs/agora/testing/src/lib/server/index.ts"],
```

Comment:
The existing convention is to have one item listed in this file per library (project). Are those two items needed?

Reply:
I'll change to a single item.

Reply:
Thanks!